src/app/ngrx/effects/routing.effects.ts

```diff
@@ -25,26 +25,14 @@ export class RoutingEffects {
   // The router reducer captures the CustomRouterNavigationAction to save routing state.
   // The CustomRouterNavigationAction is mocked by tests but is _not_ otherwise ever manually called by the application,
   // only through routing and the router-store connecting module.
-  @Effect({dispatch: false})
-  customRouterNavigation$: Observable<void> = this.actions$
+  @Effect()
```

Comment:
Yes, this always returns an action now. When I had charted on the metrics data model and added that work around for the Home link, an empty routing action was getting through, and it cleared out charted on the episode metrics. Having charted on the episode metrics was a poor decision on my part. I stuck charted into the metrics where I thought we'd use it from because I was struggling with selector composition. (For some reason when I first tried it, the selectors would fire updates constantly instead of just when there were changes.) It was not a good place to keep that state. With selector composition set up correctly, I've now been able to pull it back out and only keep charted in the router state.